doc/sphinx/source/changelog.rst

```diff
@@ -4,6 +4,103 @@ Changelog
+-  Extending documentation: broken recipe policy (`#2720 <https://github.com/ESMValGroup/ESMValTool/pull/2720>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+
+Diagnostics
+~~~~~~~~~~~
+
+-  Recipe and diagnostics for : Tebaldi et al.,ESD, 2021 (`#2052 <https://github.com/ESMValGroup/ESMValTool/pull/2052>`__) `debe-kevin <https://github.com/debe-kevin>`__
+-  Figures for IPCC AR6 WG1 Chapter 3 (Atmosphere) (`#2533 <https://github.com/ESMValGroup/ESMValTool/pull/2533>`__) `Lisa Bock <https://github.com/LisaBock>`__
+
+Observational and re-analysis dataset support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+-  Update CERES-EBAF to Ed4.1 (`#2752 <https://github.com/ESMValGroup/ESMValTool/pull/2752>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  New CMORizer for CALIPSO-ICECLOUD (`#2753 <https://github.com/ESMValGroup/ESMValTool/pull/2753>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  New CMORizer for CLOUDSAT-L2 (`#2754 <https://github.com/ESMValGroup/ESMValTool/pull/2754>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  Update MERRA2 cmorizer with extra 2D and 3D variables (`#2774 <https://github.com/ESMValGroup/ESMValTool/pull/2774>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+
+Automatic testing
+~~~~~~~~~~~~~~~~~
+
+-  Pin `netcdf4 != 1.6.1` since that is spitting large numbers of SegFaults (`#2796 <https://github.com/ESMValGroup/ESMValTool/pull/2796>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+
+Installation
+~~~~~~~~~~~~
+
+-  Increase esmvalcore version to 2.7.0 in environment files (`#2860 <https://github.com/ESMValGroup/ESMValTool/pull/2860>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+
+Improvements
+~~~~~~~~~~~~
+
+-  Fix tebaldi21esd (`#2749 <https://github.com/ESMValGroup/ESMValTool/pull/2749>`__) `Axel Lauer <https://github.com/axel-lauer>`__
+-  Initial Commit for Recipe Test Workflow (`#2787 <https://github.com/ESMValGroup/ESMValTool/pull/2787>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+-  Added option to show basic statistics in plots of `monitor/multi_datasets.py` (`#2790 <https://github.com/ESMValGroup/ESMValTool/pull/2790>`__) `Manuel Schlund <https://github.com/schlunma>`__
+-  Update configure task to use the correct paths (`#2815 <https://github.com/ESMValGroup/ESMValTool/pull/2815>`__) `Katherine Tomkins <https://github.com/KatherineTomkins>`__
+-  Update the install_cold task to install the latest version of ESMValTool and ESMValCore (`#2803 <https://github.com/ESMValGroup/ESMValTool/pull/2803>`__) `Katherine Tomkins <https://github.com/KatherineTomkins>`__
+-  Add compare task to recipe_test_workflow (`#2820 <https://github.com/ESMValGroup/ESMValTool/pull/2820>`__) `mo-tgeddes <https://github.com/mo-tgeddes>`__
+-  Remove retracted datasets from `recipe_climate_change_hotspot` (`#2854 <https://github.com/ESMValGroup/ESMValTool/pull/2854>`__) `sloosvel <https://github.com/sloosvel>`__
```

Comment:
I put a `Data` label in this PR, yet it shows under `Improvements`. Maybe the `draft_release_notes` script is missing the label?

Reply:
I think Data label is not recognized by the script, let me look into it

Reply:
yeps, unfortunately Data is not a valid label - although I would argue it should be; for now it is what it is, do you want to relabel those to another (valid) label, Saskia?

Reply:
I added the diagnostic one

Reply:
so you want them under Diagnostics?